config/data_centers/data_centers.json

```
@@ -898,5 +898,145 @@
     "status": "operational",
     "source": "https://docs.aws.amazon.com/global-infrastructure/latest/regions/aws-regions.html",
     "operationalSince": null
+  },
+  "ovh-us-hillsboro": {
+    "provider": "ovh",
+    "displayName": "Hillsboro",
+    "zoneKey": "US",
+    "lonlat": [45.5225, -122.992],
+    "region": "North America",
```

A bit unclear I'll admit but I'm pretty sure this needs to be:
```suggestion
    "region": "us-hillsboro",
```
(this applies to all of them)

The key is made up of the provider + region so the validation functions works properly.

Otherwise I think it looks good!

---

I have updated the region